{{cookiecutter.app_name}}/test/http.go

```diff
@@ -0,0 +1,22 @@
+package test
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/onsi/ginkgo"
```

Comment:
This is a slight inconsistency as in tests the import is:

 ```suggestion
	. "github.com/onsi/ginkgo"
```

So I suggest being consistent here also. IMO for tests, not having the prefix `ginkgo` should be preferred.

Reply:
Fixed in 60224dd7f364033cfda40b0fbeaa16345c505363